From 42ea178770f3b4fd8390b9498ac621eafea54fdb Mon Sep 17 00:00:00 2001
From: billy4479 <giachi.ellero@gmail.com>
Date: Fri, 2 Feb 2024 19:28:06 +0100
Subject: [PATCH 6/6] Downgrade okio to 3.0.0

This makes sure that the version of okio is the same as the version
required by okhttp 4.10.0. Upstream should probably make sure of this
themselves but oh well
---
 build.gradle.kts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build.gradle.kts b/build.gradle.kts
index c84c258..84e5b76 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -31,9 +31,9 @@ val distJarOutput by configurations.creating {
 dependencies {
 	implementation("commons-cli:commons-cli:1.5.0")
 	implementation("com.google.code.gson:gson:2.9.0")
-	implementation("com.squareup.okio:okio:3.1.0")
+	implementation("com.squareup.okio:okio:3.0.0")
 	implementation(kotlin("stdlib-jdk8"))
-	implementation("com.squareup.okhttp3:okhttp:4.10.0")
+	implementation("com.squareup.okhttp3:okhttp:4.1.0")
 	implementation("cc.ekblad:4koma:1.1.0")
 
 	r8("com.android.tools:r8:3.3.28")
-- 
2.43.0

